### ISSUE TITLE
CHASM Backend (Transformed jars) Causes Improper jar Seeking

### ISSUE THREAD

**Issue by rinOfTheStars** [CursedMC/YummyQuiltHacks#17]
As discussed in the CursedMC Discord, 0.1.2 is unable to load on the most recent stable Quilt Loader version (0.18.7), as the Loader is unable to find the jar-in-jar'd dependencies.

**Comment by rinOfTheStars** [comment]
Going to look into fixing this myself and make a pr for it if I can get it to be fixed

**Comment by sylv256** [comment]
I need to fix the codebase first

**Comment by rinOfTheStars** [comment]
I should just be able to roll back to the last commit in the 0.1.2 changelog, right?

**Comment by sylv256** [comment]
Currently, the code is for 1.0, and I didn't care to put it on a separate branch.

**Comment by sylv256** [comment]
> I should just be able to roll back to the last commit in the 0.1.2 changelog, right?

I think so. I'll make a branch.

**Comment by sylv256** [comment]
I might just make a `0.2.0-alpha.1` version first. The old codebase needed cleaning, and it'd be hard to maintain two separate versions of the codebase.

**Comment by rinOfTheStars** [comment]
Fair enough lol, I'm just trying to get 0.1.2 to work on newer loader versions for now

**Comment by rinOfTheStars** [comment]
Probably also going to make a separate issue for dropping the JIJing of that one QSL module since that breaks on older versions while everything else is pretty much version-agnostic

**Comment by sylv256** [comment]
Actually, I think all I needed to do is remove the `yqh-relaunch` language adapter from `0.2.0` and we'll be good. I'll test this out right now.

**Comment by sylv256** [comment]
Okay, the fixed version is on `dev/0.2.x`
